scipy/stats/_continuous_distns.py

```diff
@@ -4609,6 +4609,51 @@ def _entropy(self):
 semicircular = semicircular_gen(a=-1.0, b=1.0, name="semicircular")
 
 
+class skew_cauchy_gen(rv_continuous):
+    """A skewed Cauchy random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The pdf is::
+
+        skewcauchy.pdf(x, a) = 1 / (np.pi * (x**2 / (a * np.sign(x) + 1)**2 + 1))
+
+    `skewcauchy` takes ``a`` as a skewness parameter
+    When a=0 the distribution is identical to the usual Cauchy distribution.
+
+    %(after_notes)s
+
+    %(example)s
+    """
+
+    def _argcheck(self, a):
+        return np.isfinite(a)
```

Comment:
Nothing was explicitly mentioned in the sources, but it does seems to me that the distribution should be undefined for `np.abs(a) = 1`. Will make that correction.